app/views/aspects/_aspect_stream.haml

```diff
@@ -3,14 +3,16 @@
 -#   the COPYRIGHT file.
 
 - if user_signed_in? && @person != current_user.person
-  %h3#aspect_stream_header.stream_title
+  %h3.stream-title
     = stream.title
 
-= render 'publisher/publisher', publisher_aspects_for(stream)
+.container-fluid.main-stream-publisher
+  .pull-left.hidden-xs
+    = owner_image_link
+  = render "publisher/publisher", publisher_aspects_for(stream)
 
-#gs-shim{:title => popover_with_close_html("3. #{t('.stay_updated')}"), 'data-content' => t('.stay_updated_explanation')}
-
-#main_stream.stream
+.stream#main_stream{:title => popover_with_close_html("3. #{t('.stay_updated')}"),
+  "data-content" => t(".stay_updated_explanation")}
```

Comment:
Hm, that means two empty attributes, not really clean. I'm going to stay with the if / else here I think.